src/Test/Perf/util/RelativeDirectory.cs

```diff
@@ -23,27 +23,13 @@ public RelativeDirectory([CallerFilePath] string workingFile = "")
 
         public string MyWorkingDirectory => _workingDir;
 
-
-        /// Returns the directory that you can put artifacts like
-        /// etl traces or compiled binaries
-        public string MyArtifactsDirectory
-        {
-            get
-            {
-                var path = Path.Combine(MyWorkingDirectory, "artifacts");
-                Directory.CreateDirectory(path);
-                return path;
-            }
-        }
-
-        public string MyTempDirectory
+        public string TempDirectory
         {
             get
             {
-                var workingDir = MyWorkingDirectory;
-                var path = Path.Combine(workingDir, "temp");
-                Directory.CreateDirectory(path);
-                return path;
+                var tempDirectory = Path.Combine(Environment.ExpandEnvironmentVariables("%SYSTEMDRIVE%")+@"\", "PerfTemp");
```

Comment:
Re:  %USERPROFILE% vs %SYSTEMDRIVE%, ok.

Re: ```@"\"```, I see...%SYSTEMDRIVE% is just "C:".  It might be clearer to just do:

```Environment.ExpandEnvironmentVariables("%SYSTEMDRIVE%") + "\PerfTemp"```

```Path.Combine``` isn't really buying us any additional safety, because we're assuming %SYSTEMDRIVE% never ends in a path separator, and just glancing at this code, it looks like a bug to be using ```Path.Combine``` and an explicit directory separator (the whole point of ```Path.Combine``` being to avoid assumptions about directory separators.